[2022-08-09T15:52:49Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
[2022-08-09T15:52:49Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Running on http://192.168.0.197:5000/ (Press CTRL+C to quit)
[2022-08-09T15:52:49Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Restarting with stat
[2022-08-09T15:52:49Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T15:52:49Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T15:53:11Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO - 127.0.0.1 - - [09/Aug/2022 15:53:11] "POST /processes/s100/execution HTTP/1.1" 200 -
[2022-08-09T15:54:32Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T15:54:32Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T15:54:47Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
[2022-08-09T15:54:47Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Running on http://192.168.0.197:5000/ (Press CTRL+C to quit)
[2022-08-09T15:54:47Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Restarting with stat
[2022-08-09T15:54:48Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T15:54:48Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T15:55:06Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/pygeoapi/process/manager/dummy.py:91} ERROR - 'list' object has no attribute 'shape'
[2022-08-09T15:55:06Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO - 127.0.0.1 - - [09/Aug/2022 15:55:06] "POST /processes/s100/execution HTTP/1.1" 200 -
[2022-08-09T15:55:23Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T15:55:23Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T15:57:21Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
[2022-08-09T15:57:21Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Running on http://192.168.0.197:5000/ (Press CTRL+C to quit)
[2022-08-09T15:57:21Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Restarting with stat
[2022-08-09T15:57:22Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T15:57:22Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T15:58:29Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T15:58:29Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T15:58:30Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
[2022-08-09T15:58:30Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Running on http://192.168.0.197:5000/ (Press CTRL+C to quit)
[2022-08-09T15:58:30Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Restarting with stat
[2022-08-09T15:58:31Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T15:58:31Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T15:58:33Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T15:58:33Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:01:12Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
[2022-08-09T16:01:12Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Running on http://192.168.0.197:5000/ (Press CTRL+C to quit)
[2022-08-09T16:01:12Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Restarting with stat
[2022-08-09T16:01:13Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:01:13Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:01:30Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/pygeoapi/process/manager/dummy.py:91} ERROR - 0
[2022-08-09T16:01:30Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO - 127.0.0.1 - - [09/Aug/2022 16:01:30] "POST /processes/s100/execution HTTP/1.1" 200 -
[2022-08-09T16:01:49Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:01:49Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:02:12Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
[2022-08-09T16:02:12Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Running on http://192.168.0.197:5000/ (Press CTRL+C to quit)
[2022-08-09T16:02:12Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Restarting with stat
[2022-08-09T16:02:13Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:02:13Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:02:46Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/pygeoapi/process/manager/dummy.py:91} ERROR - 0
[2022-08-09T16:02:46Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO - 127.0.0.1 - - [09/Aug/2022 16:02:46] "POST /processes/s100/execution HTTP/1.1" 200 -
[2022-08-09T16:03:07Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:03:07Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:05:36Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
[2022-08-09T16:05:36Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Running on http://192.168.0.197:5000/ (Press CTRL+C to quit)
[2022-08-09T16:05:36Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Restarting with stat
[2022-08-09T16:05:37Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:05:37Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:05:38Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:05:38Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:05:42Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
[2022-08-09T16:05:42Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Running on http://192.168.0.197:5000/ (Press CTRL+C to quit)
[2022-08-09T16:05:42Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Restarting with stat
[2022-08-09T16:05:43Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:05:43Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:05:46Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:05:46Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:05:56Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
[2022-08-09T16:05:56Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Running on http://192.168.0.197:5000/ (Press CTRL+C to quit)
[2022-08-09T16:05:56Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Restarting with stat
[2022-08-09T16:05:56Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:05:56Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:05:58Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:05:58Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:06:28Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
[2022-08-09T16:06:28Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Running on http://192.168.0.197:5000/ (Press CTRL+C to quit)
[2022-08-09T16:06:28Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Restarting with stat
[2022-08-09T16:06:28Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:06:28Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:06:47Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/pygeoapi/process/manager/dummy.py:91} ERROR - 0
[2022-08-09T16:06:47Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO - 127.0.0.1 - - [09/Aug/2022 16:06:47] "POST /processes/s100/execution HTTP/1.1" 200 -
[2022-08-09T16:07:02Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:07:02Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:08:39Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
[2022-08-09T16:08:39Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Running on http://192.168.0.197:5000/ (Press CTRL+C to quit)
[2022-08-09T16:08:39Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Restarting with stat
[2022-08-09T16:08:40Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:08:40Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:08:59Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/pygeoapi/process/manager/dummy.py:91} ERROR - 0
[2022-08-09T16:08:59Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO - 127.0.0.1 - - [09/Aug/2022 16:08:59] "POST /processes/s100/execution HTTP/1.1" 200 -
[2022-08-09T16:11:24Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:11:24Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:12:05Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
[2022-08-09T16:12:05Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Running on http://192.168.0.197:5000/ (Press CTRL+C to quit)
[2022-08-09T16:12:05Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Restarting with stat
[2022-08-09T16:12:06Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:12:06Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:14:33Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/pygeoapi/process/manager/dummy.py:91} ERROR - 0
[2022-08-09T16:14:33Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO - 127.0.0.1 - - [09/Aug/2022 16:14:33] "POST /processes/s100/execution HTTP/1.1" 200 -
[2022-08-09T16:17:00Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:17:00Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:18:39Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
[2022-08-09T16:18:39Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Running on http://192.168.0.197:5000/ (Press CTRL+C to quit)
[2022-08-09T16:18:39Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Restarting with stat
[2022-08-09T16:18:40Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:18:40Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:19:22Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/pygeoapi/process/manager/dummy.py:91} ERROR - 0
[2022-08-09T16:19:22Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO - 127.0.0.1 - - [09/Aug/2022 16:19:22] "POST /processes/s100/execution HTTP/1.1" 200 -
[2022-08-09T16:20:33Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:20:33Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:21:28Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
[2022-08-09T16:21:28Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Running on http://192.168.0.197:5000/ (Press CTRL+C to quit)
[2022-08-09T16:21:28Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Restarting with stat
[2022-08-09T16:21:28Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:21:28Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:21:44Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/pygeoapi/process/manager/dummy.py:91} ERROR - 0
[2022-08-09T16:21:44Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO - 127.0.0.1 - - [09/Aug/2022 16:21:44] "POST /processes/s100/execution HTTP/1.1" 200 -
[2022-08-09T16:23:56Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:23:56Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:24:07Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
[2022-08-09T16:24:07Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Running on http://192.168.0.197:5000/ (Press CTRL+C to quit)
[2022-08-09T16:24:07Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Restarting with stat
[2022-08-09T16:24:07Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:24:07Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:24:24Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/pygeoapi/process/manager/dummy.py:91} ERROR - 0
[2022-08-09T16:24:24Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO - 127.0.0.1 - - [09/Aug/2022 16:24:24] "POST /processes/s100/execution HTTP/1.1" 200 -
[2022-08-09T16:25:30Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/pygeoapi/process/manager/dummy.py:91} ERROR - 0
[2022-08-09T16:25:30Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO - 127.0.0.1 - - [09/Aug/2022 16:25:30] "POST /processes/s100/execution HTTP/1.1" 200 -
[2022-08-09T16:29:43Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:29:43Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:30:59Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
[2022-08-09T16:30:59Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Running on http://192.168.0.197:5000/ (Press CTRL+C to quit)
[2022-08-09T16:30:59Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Restarting with stat
[2022-08-09T16:31:00Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:31:00Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:31:18Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/pygeoapi/process/manager/dummy.py:91} ERROR - Unable to create group (name already exists)
[2022-08-09T16:31:18Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO - 127.0.0.1 - - [09/Aug/2022 16:31:18] "POST /processes/s100/execution HTTP/1.1" 200 -
[2022-08-09T16:31:35Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:31:35Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:35:20Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
[2022-08-09T16:35:20Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Running on http://192.168.0.197:5000/ (Press CTRL+C to quit)
[2022-08-09T16:35:20Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Restarting with stat
[2022-08-09T16:35:21Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:35:21Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:35:42Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/pygeoapi/process/manager/dummy.py:91} ERROR - Unable to create group (name already exists)
[2022-08-09T16:35:42Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO - 127.0.0.1 - - [09/Aug/2022 16:35:42] "POST /processes/s100/execution HTTP/1.1" 200 -
[2022-08-09T16:35:57Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:35:57Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:38:51Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
[2022-08-09T16:38:51Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Running on http://192.168.0.197:5000/ (Press CTRL+C to quit)
[2022-08-09T16:38:51Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Restarting with stat
[2022-08-09T16:38:51Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:38:51Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:39:12Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/pygeoapi/process/manager/dummy.py:91} ERROR - Unable to create group (name already exists)
[2022-08-09T16:39:12Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO - 127.0.0.1 - - [09/Aug/2022 16:39:12] "POST /processes/s100/execution HTTP/1.1" 200 -
[2022-08-09T16:39:31Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:39:31Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:47:26Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
[2022-08-09T16:47:26Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Running on http://192.168.0.197:5000/ (Press CTRL+C to quit)
[2022-08-09T16:47:26Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Restarting with stat
[2022-08-09T16:47:27Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:47:27Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:47:46Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/pygeoapi/process/manager/dummy.py:91} ERROR - unsupported operand type(s) for -: 'bytes' and 'bytes'
[2022-08-09T16:47:46Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO - 127.0.0.1 - - [09/Aug/2022 16:47:46] "POST /processes/s100/execution HTTP/1.1" 200 -
[2022-08-09T16:48:08Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:48:08Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:48:40Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
[2022-08-09T16:48:40Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Running on http://192.168.0.197:5000/ (Press CTRL+C to quit)
[2022-08-09T16:48:40Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Restarting with stat
[2022-08-09T16:48:41Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:48:41Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:48:57Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/pygeoapi/process/manager/dummy.py:91} ERROR - name 'instance_group_path' is not defined
[2022-08-09T16:48:57Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO - 127.0.0.1 - - [09/Aug/2022 16:48:57] "POST /processes/s100/execution HTTP/1.1" 200 -
[2022-08-09T16:49:25Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:49:25Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:49:45Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
[2022-08-09T16:49:45Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Running on http://192.168.0.197:5000/ (Press CTRL+C to quit)
[2022-08-09T16:49:45Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Restarting with stat
[2022-08-09T16:49:45Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:49:45Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
[2022-08-09T16:50:03Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO - 127.0.0.1 - - [09/Aug/2022 16:50:03] "POST /processes/s100/execution HTTP/1.1" 200 -
[2022-08-09T16:50:28Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} WARNING -  * Debugger is active!
[2022-08-09T16:50:28Z] {/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/werkzeug/_internal.py:225} INFO -  * Debugger PIN: 414-842-819
